BackofficeBundle/Resources/public/less/editTable.less

```diff
@@ -1,7 +1,40 @@
-.back-to-list {
-    margin-bottom: 20px;
+#widget-grid {
+
+    .submit_form, .submit_form:active, .back-to-list, .back-to-list:active, .bh-button-add, .bh-button-add:active {
+        width: 119px;
+        position: fixed;
+        top: 54px !important;
+        z-index: 901;
+    }
+    .back-to-list {
+        right: 15%;
+
+    }
+    .back-to-list:active, .submit_form:active, .bh-button-add:active {
+        left: 75.7% !important;
```

Comment:
I'm not sure about this kind of size: 75,7% seems to be very linked to your personnnal screen resolution. Have you tried with some other resolutions ? Have you tried with the fullscreen widget ?